app/models/time_extension_validation_request.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+class TimeExtensionValidationRequest < ValidationRequest
+  validates :proposed_expiry_date, presence: true
+  validates :cancel_reason, presence: true, if: :cancelled?
+  validates :reason, presence: true
+  validate :rejected_reason_is_present?
+  validate :allows_only_one_open_time_extension, on: :create
+  validate :proposed_expiry_is_later?
+
+  def proposed_expiry_is_later?
+    return if planning_application.nil? || proposed_expiry_date.blank?
+
+    if proposed_expiry_date < planning_application.expiry_date
```

Comment:
Minor point, but what happens if the proposed date is the same as the current expiry date? It's not an error in the same kind of way (because going backwards isn't allowed but staying the same is fine), but it's still sort of nonsensical to request a change where the proposed is the same as the existing.

```suggestion
    if proposed_expiry_date <= planning_application.expiry_date
```